WalletWasabi.Gui/Assets/LegalDocuments.txt

```diff
@@ -33,23 +33,23 @@ We may make changes to these Terms, including when there are changes to our Serv
+We expressly declare that we do not store any private keys. Encrypted private key information is stored locally on your computer in a wallet file and/or backed up with Recovery Words, shown at the creation of the wallet. Private keys can be accessed with a password, which you have provided at the creation of the wallet. It is solely your responsibility to backup and to safekeep your private keys. In the event of your loss of private keys, you may permanently lose access to your public addresses, since we are not able to recover the private keys for You.
 If You do not understand the above paragraph please learn more about Bitcoin transactions before using our Services.
-The download and use of the Wasabi Wallet software is subject to the acceptance of our terms and conditions. From the point of downloading we do not store or manage any data, we do not even have knowledge about neither the installation, nor the use of the wallet.
+The download and use of the Wasabi Wallet software is subject to the acceptance of our terms and conditions. From the point of downloading we do not store or manage any data, and we have no knowledge or information about either the installation, or the use of the wallet.
 The client application (“Client Application”) is a free and open-source application, the sole purpose of which is to grant You access to the Bitcoin network and our Services (as defined below) without the need or requirement to create or maintain a user account. The application itself is an ordinary Bitcoin desktop wallet with the ordinary features.
-The Privacy Oriented Bitcoin Transaction Services is an online service that implements Chaumian CoinJoin to prevent third parties to spy on the Blockchain, and ZeroLink, an end-to-end wallet privacy framework to make sure no privacy leaks happen by any other means. Throughout the process the Service does not initiate or process any standalone transactions whatsoever towards third parties (i.e. non-users of the Service) and therefore does not store or transmit value belonging to others. For further information on how it works please visit our GitHub page: https://github.com/zkSNACKs.
+The Privacy Oriented Bitcoin Transaction Services is an online service that implements Chaumian CoinJoin to prevent third parties from spying on the Blockchain, and ZeroLink, an end-to-end wallet privacy framework to make sure no privacy leaks happen by any other means. Throughout the process the Service does not initiate or process any standalone transactions whatsoever towards third parties (i.e. non-users of the Service) and therefore does not store or transmit value belonging to others. For further information on how it works please visit our GitHub page: https://github.com/zkSNACKs.
```

Comment:
GitHub shows that something is changed here, but I cannot tell what exactly...

Reply:
Before:
Chaumian CoinJoin to prevent third parties **to spy** on the Blockchain
After:
Chaumian CoinJoin to prevent third parties **from spying** on the Blockchain